parm/aero/berror/aero_diagb.yaml.j2

```
@@ -53,6 +60,8 @@ variables:
   - mass_fraction_of_sea_salt004_in_air
 
 rescale: 2.0    # rescales the filtered std. dev. by "rescale"
+diagb_weight: 0.5
```

last nitpick (I swear!) - 
```suggestion
diagb weight: 0.5
```

---

> last nitpick (I swear!) -

In chem_diagb.h,  the yaml read is:

fullConfig.get("diagb_weight", diagb_weight);

We need to keep the variable as diagb_weight, so would it work to do:

fullConfig.get("diagb weight", diagb_weight):   ?

Just looks a little awkward to me, and might lead to some confusion perhaps.

---

I mainly was leaning towards spaces because other things like `number of halo points` already exist. But if you think it'll be confusing, we can keep the underscores.